File: shoring/desk/src/main/java/com/panopset/marin/oldswpkg/games/blackjack/BlackjackCmdBinder.java
package com.panopset.marin.oldswpkg.games.blackjack;

import static com.panopset.blackjackEngine.CommandDefinitions.CMD_AUTO;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_COUNT;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_DEAL;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_DECREASE;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_DOUBLE;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_HIT;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_INCREASE;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_RESET;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_SHUFFLE;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_SPLIT;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_STAND;
import static com.panopset.blackjackEngine.CommandDefinitions.CMD_SURRENDER;

import com.panopset.marin.compat.util.CommandBinder;

public class BlackjackCmdBinder extends CommandBinder {
  public BlackjackCmdBinder() {
    super();
    registerCommand(new CommandBinder.Command("Deal", CMD_DEAL, 0));
    registerCommand(new CommandBinder.Command("Hit", CMD_HIT, 0));
    registerCommand(new CommandBinder.Command("Surrender", CMD_SURRENDER, 0));
    registerCommand(new CommandBinder.Command("Stand", CMD_STAND, 0));
    registerCommand(new CommandBinder.Command("Split", CMD_SPLIT, 0));
    registerCommand(new CommandBinder.Command("Double", CMD_DOUBLE, 0));
    registerCommand(new CommandBinder.Command("Increase", CMD_INCREASE, 0));
    registerCommand(new CommandBinder.Command("Decrease", CMD_DECREASE, 0));
    registerCommand(new CommandBinder.Command("Reset", CMD_RESET, 0));
    registerCommand(new CommandBinder.Command("Shuffle", CMD_SHUFFLE, 0));
    registerCommand(new CommandBinder.Command("Auto", CMD_AUTO, 0));
    registerCommand(new CommandBinder.Command("Count", CMD_COUNT, 0));
  }
}

File: code/shoring/desk/src/main/kotlin/com/panopset/marin/oldswpkg/games/blackjack/BlackjackCmdBinder.kt
package com.panopset.marin.oldswpkg.games.blackjack

import com.panopset.blackjackEngine.*
import com.panopset.marin.compat.util.CommandBinder
import com.panopset.marin.compat.util.PanCmd

class BlackjackCmdBinder : CommandBinder() {
    init {
        registerCommand(PanCmd("Deal", CMD_DEAL, 0))
        registerCommand(PanCmd("Hit", CMD_HIT, 0))
        registerCommand(PanCmd("Surrender", CMD_SURRENDER, 0))
        registerCommand(PanCmd("Stand", CMD_STAND, 0))
        registerCommand(PanCmd("Split", CMD_SPLIT, 0))
        registerCommand(PanCmd("Double", CMD_DOUBLE, 0))
        registerCommand(PanCmd("Increase", CMD_INCREASE, 0))
        registerCommand(PanCmd("Decrease", CMD_DECREASE, 0))
        registerCommand(PanCmd("Reset", CMD_RESET, 0))
        registerCommand(PanCmd("Shuffle", CMD_SHUFFLE, 0))
        registerCommand(PanCmd("Auto", CMD_AUTO, 0))
        registerCommand(PanCmd("Count", CMD_COUNT, 0))
    }
}
